example/apparmor.txt

```diff
@@ -0,0 +1,53 @@
+#include <tunables/global>
+
+profile example flags=(attach_disconnected,mediate_deleted) {
+  #include <abstractions/base>
+
+  network,
+  deny network raw,
+
+  signal (send) set=(kill,term,int,hup,cont),
+
+  capability net_admin,
+  capability net_bind_service,
+  capability dac_read_search,
+  capability dac_override,
+
+  /bin/** ix,
+  /usr/bin/** ix,
+  /usr/lib/bashio/** ix,
+  /etc/s6/** ix,
+  /run/{s6,s6-rc*,service}/** ix,
```

Comment:
Is that not more like:
```suggestion
  /run/s6/** ix,
  /etc/{s6,s6-rc*,service}/** ix,
```

Reply:
All those folders were in /run when I was poking through the addon.  And that's what kept showing up in the audit when I put the profile in complain mode.  I'm actually not sure if s6 tries to run anything out of /etc anymore, think it copies it everything to /run first